core/state/trie_prefetcher.go

```diff
@@ -126,6 +126,9 @@ func (p *triePrefetcher) copy() *triePrefetcher {
 	// If the prefetcher is already a copy, duplicate the data
 	if p.fetches != nil {
 		for root, fetch := range p.fetches {
+			if fetch == nil {
```

Comment:
Is this an expected case?

Reply:
we had a panic previously on teku-geth-1.

if we failed to open a storage trie, then miner copies the state and then copy again(copy twice I guess), will trigger this panic.